### ISSUE TITLE
Symbol’s function definition is void: elsa-form-int-p

### ISSUE THREAD

**Issue by xuchunyang** [emacs-elsa/Elsa#142]
I tried to run elsa on elsa.el but got the error:

```
~/src/Elsa $ EMACSLOADPATH=/Users/xcy/.emacs.d/elpa-26.1.91/trinary-20180904.2313/:/Users/xcy/src/racket-mode/:/Users/xcy/.emacs.d/elpa-26.1.91/f-20190109.906/:/Users/xcy/.emacs.d/elpa-26.1.91/dash-20190128.1920/:$PWD: \
> bin/elsa elsa.el
Symbol’s function definition is void: elsa-form-int-p
```

there is a function `elsa-form-integer-p` when elsa.el is loaded.

**Comment by xuchunyang** [comment]
By the way, I didn't byte compile elsa because of the Invalid slot name error.

**Comment by Fuco1** [comment]
Yes it should most likely be `elsa-form-integer-p` indeed.

**Comment by Fuco1** [comment]
Fixed in b9bfbde867679f0c18e24e259d2f3e5a7417e492
